modules/styling/stories/Welcome.stories.mdx

```diff
@@ -0,0 +1,68 @@
+<Meta title="Features/Styling/Welcome" />
+
+# Canvas Styling
+
+This package contains everything needed to create CSS styling. This styling package contains a
+runtime for development and a static parsing process for build time.
+
+Here are the goals for this project:
+
+- TypeScript autocomplete of CSS object properties
+- Low runtime for development
+- Static CSS compilation for faster user experience
+- Static CSS extraction for CSS only packages
+- Dynamic styles using CSS Variables
+
+If you're using Canvas Kit and not directly using this package, there is nothing extra to do on your
+end. The Canvas Kit packages are using the static compilation as part of the build process. If you
+want to use this package for your own styles, you don't need to do anything special to use in
+development. Included is a small runtime to get styling working. If you wish to statically compile
+your CSS from your TypeScript files for faster page loading, visit the
+[Getting Started](/docs/features-styling-getting-started--page)
+
+## What is this?
+
+Canvas Kit Styling is two things:
```

Comment:
I'm wondering if we should link out to the discussion of the future of css and why we built this?

Reply:
yeah it seems like this is built BECAUSE we want to better support theming/ and styling and move away from emotion

Reply:
I'm adding a "Why" section with a link to the discussion.